experimental/BasisLieHighestWeight/src/UserFunctions.jl

```diff
@@ -125,8 +125,9 @@ function basis_lie_highest_weight(
   type::Symbol, rank::Int, highest_weight::Vector{Int}; monomial_ordering::Symbol=:degrevlex
 )
   L = lie_algebra(QQ, type, rank)
+  V = SimpleModuleData(L, WeightLatticeElem(root_system(L), highest_weight))
```

Comment:
could you instead add a constructor `SimpleModuleData(::LieAlgebra, ::Vector{<:IntegerUnion})` that takes care of the wrapping in `WeightLatticeElem` and then replace this line by
```suggestion
  V = SimpleModuleData(L, highest_weight)
```
and similar for all other places in this file (same for `DemazureModuleData`)?